setup.py

```diff
@@ -144,6 +160,6 @@ def write_version_py(filename=None):
       install_requires=INSTALL_REQUIRES,
       tests_require=TESTS_REQUIRE,
       url=URL,
-      test_suite='nose.collector',
+      cmdclass={'test': PyTest},
```

Comment:
This is cute -- but do we even need it? I don't actually use `python setup.py test`.